snowflake/cmd/deleteBucket_test.go

```diff
@@ -44,13 +44,13 @@ func TestDeleteS3Objects(t *testing.T) {
 		mockS3Client := s3clienttesting.NewMockInterface(ctrl)
 		k := ""
 		output := s3.ListObjectsV2Output{
-			Contents: []s3types.Object{s3types.Object{Key: &k}},
+			Contents: []s3types.Object{{Key: &k}},
```

Comment:
Why do we also have changes in snowflake files in this commit?

Reply:
Ack